packages/tasks/src/model-libraries.ts

```diff
@@ -446,6 +446,13 @@ export const MODEL_LIBRARIES_UI_ELEMENTS = {
 		snippets: snippets.pyannote_audio,
 		filter: true,
 	},
+	"py-feat": {
+		prettyLabel: "Py-Feat",
+		repoName: "py-feat",
+		repoUrl: "https://github.com/cosanlab/py-feat",
+		docsUrl: "https://py-feat.org/",
+		filter: true,
```

Comment:
Marking `filter: true,` will add the library in the left sidebar on https://huggingface.co/models. This is usually reserved to popular libraries with at least hundreds of models as we want to keep the list short. 

![image](https://github.com/user-attachments/assets/10ba2798-23a9-4a6a-90d9-1398769f0254)

```suggestion
		filter: false,
```

Setting `filter: false` don't mean that it'll not be possible to filter by this library, it just means that it'll not be listed on this page. It's still possible to filter by `py-feat` by going to a py feat model and clicking on the tag.